back-end/src/main/java/admin/member/ui/MemberController.java

```diff
@@ -0,0 +1,63 @@
+package admin.member.ui;
+
+import admin.member.application.MemberService;
+import admin.member.dto.request.ChangeLabRequest;
+import admin.member.dto.request.MemberInfoRequest;
+import admin.member.dto.request.MemberRequest;
+import admin.member.dto.request.MemberTypeRequest;
+import admin.member.dto.response.MemberResponse;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+
+@RestController
+@RequestMapping("/api/members")
```

Comment:
```suggestion
@RequestMapping("/api/members/")
```
`/`가 빠져도 잘 동작하는 것 같네요! 그래도 path를 통일시켜주면 좋을 것 같아요!ㅎㅎㅎ (제안해봅니당..)

Reply:
안 붙이는 걸로 통일하는건 어떨까요?

Reply:
에드 의견따라 각각의 Mapping 앞에 '/' 를 붙여주도록 하겠습니다!